.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 npm-debug.log
 .DS_Store
 dist
+yarn.lock
```

Comment:
http://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

although they claim if you are a lib author, then you should not commit the `yarn.lock` but the only reason for not committing the file is that it's redundant when you install the library. But still, it makes a huge sense for development.